pkg/api/mlflow/service/artifact/validators_test.go

```diff
@@ -139,3 +139,156 @@ func TestValidateListArtifactsRequest_Error(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateGetArtifactRequest_Ok(t *testing.T) {
+	testData := []struct {
+		name    string
+		request request.GetArtifactRequest
+	}{
+		{
+			name: "NotEmptyPathCase1",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "foo/..asd../",
+			},
+		},
+		{
+			name: "NotEmptyPathCase2",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "./foo",
+			},
+		},
+		{
+			name: "NotEmptyPathCase3",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "./foo/",
+			},
+		},
+		{
+			name: "NotEmptyPathCase4",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  ".foo",
+			},
+		},
+		{
+			name: "NotEmptyPathCase5",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "foo.bar",
+			},
+		},
+		{
+			name: "NotEmptyPathCase6",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "foo..bar",
+			},
+		},
+		{
+			name: "NotEmptyPathCase7",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "foo../bar",
+			},
+		},
+		{
+			name: "NotEmptyPathCase8",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "foo/..bar",
+			},
+		},
+		{
+			name: "FilenameNoSlash",
+			request: request.GetArtifactRequest{
+				RunID: "run_id",
+				Path:  "foo.txt",
+			},
+		},
+		{
+			name: "Dirname/Filename",
+			request: request.GetArtifactRequest{
+				RunID: "directory/run_id",
+				Path:  "foo.txt",
+			},
+		},
+	}
+
+	for _, tt := range testData {
+		t.Run(tt.name, func(t *testing.T) {
+			assert.Nil(t, ValidateGetArtifactRequest(&tt.request))
+		})
+	}
+}
+
+func TestValidateGetArtifactRequest_Error(t *testing.T) {
+	testData := []struct {
```

Comment:
sometimes we use `testCases` sometimes `testData`, can you please unity it? for me `tests` just works really well.

Reply:
done